renku/core/commands/checks/migration.py

```diff
@@ -31,7 +31,10 @@
 
 def dataset_pre_0_3(client):
     """Return paths of dataset metadata for pre 0.3.4."""
-    return (client.path / 'data').rglob(client.METADATA)
+    project_is_pre_0_3 = int(client.project.version) < 2
```

Comment:
@ciyer @mohammad-sdsc Small head's up here, just to avoid stepping on each other's toes. Since both of you are editing this, I think we should have both fixtures to ensure that everything works in the end.